tests/integration/ha_tests/test_self_healing.py

```diff
@@ -173,8 +171,14 @@ async def test_freeze_db_process(
 
 @pytest.mark.parametrize("process", DB_PROCESSES)
 async def test_restart_db_process(
-    ops_test: OpsTest, process: str, continuous_writes, master_start_timeout
+    ops_test: OpsTest, process: str, continuous_writes, primary_start_timeout
 ) -> None:
+    # Set signal based on the process
+    if process == PATRONI_PROCESS:
+        signal = "SIGTERM"
+    else:
+        signal = "SIGINT"
```

Comment:
Pkill was likely terminating the wrong process before. SIGTEMing postgres does not terminate until patroni is also down, SIGINTing patroni was sometimes interpreted as keyboard interupt in `pysyncobj` library